packages/@ama-sdk/schematics/collection.json

```diff
@@ -31,6 +31,10 @@
       "description": "Generate a Java SDK source code based on swagger specification",
       "factory": "./schematics/java/client-core/index#ngGenerateJavaClientCore",
       "schema": "./schematics/java/client-core/schema.json"
+    },
+    "migration-v10_3": {
```

Comment:
I wonder if, only for the manual updates, it would not make more sense to expose a single `migrate` schematics taking the 2 `--from` `--to` arguments to determine which rules to apply

Reply:
I thinking in user and renovate case.
How you will document for the user and indicate to renovate to run this schematic?
I case you have 1 schematic that takes the version as input you will need to just add in the doc this single schematic and the run of it can be added within the shared Renovate preset

Reply:
This

Reply:
After discussion, we agreed to exclude this script for now as we do not have a proper schematic to manage the migration for non angular project. We should first implement a schematic

Note: meanwhile we will need to document the manual migration step